pkg/syncer/spec/spec_controller.go

```diff
@@ -60,6 +60,7 @@ func NewSpecSyncer(gvrs []schema.GroupVersionResource, upstreamClusterName logic
 	for _, mutator := range []specmutators.Mutator{
 		specmutators.NewDeploymentMutator(upstreamURL),
 		specmutators.NewSecretMutator(),
+		specmutators.NewConfigMapMutator(),
```

Comment:
would pretty much prefer per-subject mutators, rather than having them by kind. This will quickly become hard to understand.